frontend/src/APIClients/LogRecordAPIClient.ts

```diff
@@ -47,7 +47,7 @@ const countLogRecords = async ({
 };
 
 const filterLogRecords = async ({
-  building = "",
+  buildingId,
```

Comment:
after the prev changes this can be `buildingId = []` (you'll have to change the param type)